src/charts/line.js

```diff
@@ -711,6 +711,7 @@ define(function(require){
             topicLine = d3Shape.line()
                 .curve(curveMap[lineCurve])
                 .x(({date}) => xScale(date))
+                .defined(({value}) => value !== null)
```

Comment:
```
.defined(({value}) => value)
```
should also do the trick

Reply:
Wouldn't be instead
``` .defined(({value}) => !!value)```

Reply:
(sorry, misclick)

Reply:
not if value is 0 which is a valid number but will produce false

Reply:
True! Let's leave it like this then